packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/converter/convertDiagram.ts

```diff
@@ -65,25 +65,25 @@ const toRectangularNode = (gqlNode: GQLNode, gqlParentNode: GQLNode | null): Nod
   const horizontalAliment = gqlNode.label.type.substring(horitonzalAlignmentIndex + '-h_'.length);
 
   if (verticalAlignment === 'top') {
-    data.label.style.alignSelf = 'flex-start';
+    data.label!.style.alignSelf = 'flex-start';
```

Comment:
Test for null instead using something like:

```
if (data.label) {
  // all the code should go in here
}
```

Your code is the equivalent of `Optional.ofNullable(...).get()` without a check for null. This comment applies to every single instance of the `!.` operator (you could pretty much add a check to ban this operator altogether)

Reply:
Done